### PR TITLE
Add prjxray conda package

### PR DESCRIPTION
**Pull request by acomodi**
Signed-off-by: Alessandro Comodi <acomodi@antmicro.com>

### CODE REVIEW

File: prjxray/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  string: {{ environ.get('DATE_STR') }}
+  script_env:
+    - CI
+    - TRAVIS
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
```
Comment (mithro) [inline prjxray/meta.yaml:24]: Does prjxray actually need all these dependencies?

**Comment by mithro** [comment]
Contents looks good, but I think it does highlight that a bunch of these tools have very generic names -- IE `bitread` / `bittool` / etc -- It is likely these names will conflict with other tools and we should fix them.
```
bin/bitread
bin/bittool
bin/frame_address_decoder
bin/gen_part_base_yaml
bin/segmatch
bin/xc7frames2bit
bin/xc7patch
info/about.json
info/files
info/git
info/hash_input.json
info/index.json
info/licenses/COPYING
info/paths.json
info/recipe/build.sh
info/recipe/conda_build_config.yaml
info/recipe/condarc
info/recipe/meta.yaml
info/recipe/meta.yaml.template
info/recipe/recipe_append.yaml
```

**Comment by acomodi** [comment]
@mithro bitread and bittool should be the only ones that may create conflicts. What about xc7bitread and xc7bittool (similarly to xc7frames2bit?

**Comment by mithro** [comment]
Probably all of them should start with `xc7`?
```
bin/xc7_frame_address_decoder
bin/xc7_gen_part_base_yaml
bin/xc7_segmatch
```

**Comment by acomodi** [comment]
@mithro Ok, we should probably also change all the code that depends on these tools in prjxray and the other repos

**Comment by mithro** [comment]
@acomodi Do you want to log an issue about that and we can still merge this one?

**Comment by acomodi** [comment]
@mithro Yeah, that sounds good to me

**Comment by acomodi** [comment]
@mithro Opened an issue about the xc7 prefix: https://github.com/SymbiFlow/prjxray/issues/1343

I will go ahead and merge this one.

**Pull request merged**